schema/divisions/division.yaml

```diff
@@ -195,4 +195,12 @@ properties:     # JSON Schema: Top-level object properties.
         minItems: 1
         uniqueItems: true
         items: { "$ref": "../defs.yaml#/$defs/propertyDefinitions/id" }
+      capital_of_divisions:
+        description:
+          Division IDs and subtypes of division's this division is capital of. If present,
+          this property will refer to the division IDs of the countries, regions, counties, etc.
```

Comment:
1. [  ] Minor: Can you drop the apostrophe in *division's* on the first line? The apostrophe indicates possession, whereas you want plural.
2. [ ] Minor: Can we insert an *a* between "capital" and "of" on the first line?
3. [ ] Minor: I would suggest dropping the second sentence. It doesn't add much and seems to trail off.

Can we change the description to just:

> Division IDs and subtypes of divisions this division is a capital of.

Reply:
I posted an update with this change.